README.pt.md

```diff
@@ -1,197 +1,197 @@
 
-Acessar o mariadb-client, copiar e colar os comandos ou rodar o script.
+Access mariadb-client, copy and paste the commands or run the script.
 
-No debian: `sudo mysql`
+In debian: `sudo mysql`
 
-    MariaDB [(none)]> SOURCE doc/mysql.sql;
+    MariaDB [(none)]> SOURCE doc / mysql.sql;
     Query OK, 1 row affected (0.001 sec)
-    
+
     Query OK, 0 rows affected (0.001 sec)
-    
+
     MariaDB [(none)]> SHOW DATABASES;
-    +--------------------+
-    | Database           |
-    +--------------------+
-    | goworking          |
+    + -------------------- +
+    | Database |
+    + -------------------- +
+    | goworking |
     | information_schema |
-    | mysql              |
+    | mysql |
     | performance_schema |
-    +--------------------+
+    + -------------------- +
     4 rows in set (0.001 sec)
 
-Se o que tiver disponível for um PHPMyAdmin ou coisa parecida, descobrir como é 
-o jeito difícil de colar estes comandos simples nesse sistema complicado.  
+If what you have available is a PHPMyAdmin or something, find out how it is
+the hard way to paste these simple commands into this complicated system.
 
 #### flask-migrate
 
-    user@server:goworking-mesas$ pipenv run flask db init
-    Loading .env environment variables...
-    user@server:goworking-mesas$ pipenv run flask db upgrade
-    Loading .env environment variables...
-    INFO  [alembic.runtime.migration] Context impl MySQLImpl.
-    INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
-    INFO  [alembic.runtime.migration] Running upgrade  -> fead50b08d21, empty message
-    INFO  [alembic.runtime.migration] Running upgrade fead50b08d21 -> dc7d560eee94, empty message
+    user @ server: goworking-tables $ pipenv run flask db init
+    Loading .env environment variables ...
+    user @ server: goworking-tables $ pipenv run flask db upgrade
+    Loading .env environment variables ...
+    INFO [alembic.runtime.migration] Context impl MySQLImpl.
+    INFO [alembic.runtime.migration] Will takes over non-transactional DDL.
+    INFO [alembic.runtime.migration] Running upgrade -> fead50b08d21, empty message
+    INFO [alembic.runtime.migration] Running upgrade fead50b08d21 -> dc7d560eee94, empty message
 
 ### gunicorn
 
-Eu uso systemd.  
+I use systemd.
 
-Tem um arquivo do systemd de exemplo em *doc/gunicorn-goworking.service*.  
+There is an example systemd file in * doc / gunicorn-goworking.service * .
 
-    user@server:goworking-mesas$ sudo cp gunicorn-goworking.service /lib/systemd/system/
-    user@server:goworking-mesas$ sudo systemctl enable gunicorn-goworking.service
-    user@server:goworking-mesas$ sudo systemctl start gunicorn-goworking.service
+    user @ server: goworking-tables $ sudo cp gunicorn-goworking.service / lib / systemd / system /
+    user @ server: goworking-tables $ sudo systemctl enable gunicorn-goworking.service
+    user @ server: goworking-tables $ sudo systemctl start gunicorn-goworking.service
```

Comment:
These are standard error messages from the shell and shall not be reformatted / translated.